tests/test_task_api.py

```diff
@@ -371,3 +371,65 @@ async def test_task_collection_api_failure(
         assert data["status"] == "fail"
         assert data["log"]  # This is because of verbose=True
         assert "fail" in data["log"]
+
+
+async def test_non_python_task(
```

Comment:
We are not really testing the task API here, but rather the workflow execution.
Let's move this test to a new file, named e.g. `test_full_workflow_bash_task.py`. Or we can also include it in `test_full_workflow.py`.